contrib/gitian-descriptors/gitian-win-signer.yml

```diff
@@ -8,6 +8,7 @@ architectures:
 packages:
 - "libssl-dev"
 - "autoconf"
+- "automake"
```

Comment:
nit: `automake` depends on `autoconf`, therefore, the latter could be dropped.

Reply:
meh.